doc/users.md

```diff
@@ -1 +1,1859 @@
+        "company_code": "WARI",
+        "company_name": "PT. WAHANAARTHA RITELINDO",
+        "region_id": "5fb5e66d-e47c-47fd-b4bf-4064cf944ef7", 
+        "region_code": "WARI-03", 
+        "created": "2020-12-02 09:03:44",
+        "modified": "2020-12-02 09:03:44"
+    }
+}
+```
+
+#### Case : Positive Case 3
+
+Request Payload :
+```
+{
+    "name": "dennis",
+    "username": "vdennis",  
+    "email": "vdennis@cdc.id",  
+    "type_akses": 3,
+    "company_id": "5fb5df66-5f80-416d-a055-401bcf944ef7",
+    "branch_id": "5fb5e437-69dc-4eb6-8f8c-4022cf944ef7"
+}
+```
+
+Response HTTP Status Code : 201
+
+Response Payload :
+```
+{
+    "status_code": "CDC-201",
+    "status_message": "Created",
+    "data": {
+        "id":"1",
+        "name": "dennis",  
+        "username": "vdennis",  
+        "password": "7f2994ecd8c1498cd0661589df5051f096fe1aca",  
+        "email": "vdennis@cdc.id", 
+        "type_akses": 3,  
+        "company_id": "5fb5df66-5f80-416d-a055-401bcf944ef7",
+        "company_code": "WARI",
+        "company_name": "PT. WAHANAARTHA RITELINDO", 
+        "branch_id": "5fb5e437-69dc-4eb6-8f8c-4022cf944ef7",  
+        "branch_code": "77777",  
+        "branch_name": "WARI TEST 2", 
+        "created": "2020-12-02 09:03:44",
+        "modified": "2020-12-02 09:03:44"
+    }
+}
+```
+
+#### Case : Positive Case 4
+
+Request Payload :
+```
+{
+    "name": "dennis",
+    "username": "vdennis",  
+    "email": "vdennis@cdc.id",  
+    "type_akses": 4,
+    "company_id": "5fb5df66-5f80-416d-a055-401bcf944ef7",
+    "branch_id": "5fb5e437-69dc-4eb6-8f8c-4022cf944ef7",  
+    "kios_id": "5fb5e626-65ac-473d-af2c-3ff2cf944ef7" 
+}
+```
+
+Response HTTP Status Code : 201
+
+Response Payload :
+```
+{
+    "status_code": "CDC-201",
+    "status_message": "Created",
+    "data": {
+        "id":"1",
+        "name": "dennis",  
+        "username": "vdennis",  
+        "password": "7f2994ecd8c1498cd0661589df5051f096fe1aca",  
+        "email": "vdennis@cdc.id", 
+        "type_akses": 4,  
+        "company_id": "5fb5df66-5f80-416d-a055-401bcf944ef7",
+        "company_code": "WARI",
+        "company_name": "PT. WAHANAARTHA RITELINDO",
+        "region_id": "5fb5e66d-e47c-47fd-b4bf-4064cf944ef7", 
+        "region_code": "WARI-03", 
+        "branch_id": "5fb5e437-69dc-4eb6-8f8c-4022cf944ef7",  
+        "branch_code": "77777",  
+        "branch_name": "WARI TEST 2",  
+        "kios_id": "5fb5e626-65ac-473d-af2c-3ff2cf944ef7", 
+        "kios_code": "77777",  
+        "created": "2020-12-02 09:03:44",
+        "modified": "2020-12-02 09:03:44"
+    }
+}
+```
+
+## <a name="edit"></a>Edit Data User
+
+### Endpoint 
+PUT /user/:id
+
+### Headers
+Key | Value 
+--- | ---
+Content-Type | application/json
+Accept | application/json
+Email | admin@cdc.id
+
+### Request Payloads
+Name | Type | Example Value
+--- | --- | ---
+name | string | dennis  
+username | string | vdennis  
```

Comment:
there is no update username. if you want, please consider to create special endpoint PUT /users/change-username, or handle by auth module PUT /change-username